makefile

```diff
@@ -13,12 +13,19 @@ IMPORTS:=-Ilibdparse/src\
 
 FLAGS:=-O -gc -release -inline # keep -inline; not having it triggers an optimizer bug as pf 2.066
 
-all: $(SRC)
+hmod: $(SRC)
 	dmd $(SRC) $(IMPORTS) $(FLAGS) -ofbin/hmod
 	rm -f bin/*.o
 
+all: hmod pkg
```

Comment:
This should be the first defined target, at least if you want it to be the default goal if you type `make`.

Reply:
Ah, good point, thanks.